src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,187 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Represents the user interface for the Duke program, providing methods for
+ * displaying messages and information to the user.
+ */
+public class Ui {
+
+    /**
+     * Displays the Duke logo and an initial message to the user.
+     */
+    public void message() {
+        // Displaying Duke logo and initial message
+        String logo = " KASSIM ";
+        System.out.println("YOO I AM " + logo);
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Displays an error message encountered during the Duke program execution.
+     *
+     * @param e The DukeException containing the error message.
+     */
+    public void errorEncounter(DukeException e) {
+        System.out.println("------------------------------------------------------------");
+        System.out.println(e.getMessage());
+        System.out.println("------------------------------------------------------------");
+    }
+
+    /**
+     * Returns a message indicating that a task description is empty.
+     *
+     * @return The error message for an empty task description.
+     */
+    public String emptyErrorMessage() {
```

Comment:
Verbs should be used for method names so maybe you could change it to sound like a verb? (for e.g. `displayEmptyErrorMessage()`). Same for your other methods.

Reply:
okay, I have changed it